frontend/src/scenes/data-management/ingestion-warnings/IngestionWarningsView.tsx

```diff
@@ -17,14 +17,16 @@ export const scene: SceneExport = {
 
 const WARNING_TYPE_TO_DESCRIPTION = {
     cannot_merge_already_identified: 'Refused to merge an already identified user via $identify or $create_alias call',
+    cannot_merge_with_illegal_distinct_id:
+        'Refused to merge with an illegal distinct_id via $identify or $create_alias call',
+    cannot_merge_attepts_exhausted: 'Merge via $identify or $create_alias call failed',
```

Comment:
The purpose of this page is to give our end users **actionable** information right? I think there's no way for us to give actionable information for this erorr.

Other errors should be handled on a logging basis and go to sentry for us to handle.

Reply:
Fair point, will remove.